### ISSUE TITLE
The item isn't selected when hovering on the item on high contrast on mode

### ISSUE THREAD

**Issue by Helen09** [microsoft/pxt-ev3#653]
**Repro steps:**
• Open https://makecode.mindstorms.com/beta#editor
• Click "More..." -> "High Contrast On" to open high contrast mode
• Click "Help" or "More..." and then hover on the items under "Help" & "More..."
**Expected result:**
There should be show the item is selected when hovering on the item 

**Actual result:**
The item isn't selected when hovering on the item on high contrast on mode
![123](https://user-images.githubusercontent.com/23466737/40530361-fdf116fa-5fe7-11e8-8edb-7f4260632fc3.gif)
